test/test.go

```diff
@@ -505,3 +505,15 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) GetLatestTag(repo string) string {
+	repoPath := h.Path("pkg/dep/sources/https---" + strings.Replace(repo, "/", "-", -1))
```

Comment:
Actually, just going to create a repo with a fixed version and no other dependencies and use that